src/main/scala/com/fulcrumgenomics/alignment/NeedlemanWunschAligner.scala

```diff
@@ -25,21 +25,36 @@
 package com.fulcrumgenomics.alignment
 
 import com.fulcrumgenomics.FgBioDef._
+import com.fulcrumgenomics.alignment.NeedlemanWunschAligner.Mode.{Global, Glocal}
 import com.fulcrumgenomics.alignment.NeedlemanWunschAligner._
+import enumeratum.EnumEntry
 import htsjdk.samtools.CigarOperator
 
-import scala.collection.mutable
+import scala.collection.{immutable, mutable}
 import scala.math.max
 
 object NeedlemanWunschAligner {
+
```

Comment:
Not to be too pedantic, but having a glocal mode makes it no longer Needleman-Wunsch.

Reply:
Yeah, it's true.  But renaming seems like a big hassle.